zerver/lib/event_schema.py

```diff
@@ -1028,6 +1030,9 @@ def check_realm_update_dict(
             ("user_id", int),
             ("role", EnumType(UserProfile.ROLE_TYPES)),
         ],
+        optional_keys=[
+            ("bot_type", EnumType(UserProfile.BOT_TYPES))
+        ]
```

Comment:
These changes should not be here, I included them here by mistake. This updates event schema for `realm_user` which in no way relates to the intention of the commit. In this commit I have updated schema for `realm_user`. I will update this now.